aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -521,7 +521,7 @@ public AwsProperties() {
     this.s3DeleteTags = Sets.newHashSet();
     this.s3FileIoDeleteThreads = Runtime.getRuntime().availableProcessors();
     this.isS3DeleteEnabled = S3_DELETE_ENABLED_DEFAULT;
-    this.s3BucketToAccessPointMapping = ImmutableMap.of();
+    this.s3BucketToAccessPointMapping = Collections.emptyMap();
```

Comment:
it looks like in `S3FileIO` we're using `SerializableMap` to solve the issue with Kryo and I'm wondering whether we should do the same here

Reply:
That should work too, but I'd prefer `Collections.emptyMap()` because it is much simpler implementation and there's actually no state to serialize or copy.